### PR TITLE
Fix not being able to install Lens metrics

### PR DESCRIPTION
**Pull request by Nokel81**
Signed-off-by: Sebastian Malton <sebastian@malton.name>

### CODE REVIEW

File: src/main/resource-applier/resource-applier.ts [hunk 1]
```diff
@@ -129,9 +129,9 @@ export class ResourceApplier {
 
     const args = [
       subCmd,
-      "--kubeconfig", `"${proxyKubeconfigPath}"`,
+      "--kubeconfig", proxyKubeconfigPath,
       ...parentArgs,
-      "-f", `"${tmpDir}"`,
+      "-f", tmpDir,
```
Comment (jim-docker) [inline src/main/resource-applier/resource-applier.ts:134]: I guess this explains this when I try to uninstall Lens metrics? (Just confirmed it)
Screen Shot 2022-11-17 at 9 31 36 AM

**Review by jim-docker — APPROVED** [review]
this would impact anything that uses resource-applier, not just Lens Metrics

File: src/main/resource-applier/resource-applier.ts (hunk 1 above)
Comment (Nokel81) [reply src/main/resource-applier/resource-applier.ts:134]: It does yes

**Comment by Nokel81** [comment]
> this would impact anything that uses resource-applier, not just Lens Metrics

Fair, though I think the most common case would be this

**Pull request merged**